### PR TITLE
Batch Generator for VedaStream and VedaStream will return labels as array instead of list 

### PR DESCRIPTION
**Pull request by martham93**
Added 'batch_generator` method for `VedaBase` so it can be called like it is on a `VedaBase`. Also in order to match what is returned from the  `VedaBase` class and for the generator to work,  `VedaStream` will now return labels from the `labels` `property` as arrays. Previously they were returned as a `list.`

### CODE REVIEW

File: pyveda/vedaset/stream/vedastream.py [hunk 1]
```diff
@@ -122,7 +126,10 @@ def labels(self):
             lbls, _ = zip(*self._vset._buf)
         except ValueError:
             lbls = []
-        return BufferedVariableArray(lbls)
+        return BufferedVariableArray(np.array(lbls))
+
+    def batch_generator(self, batch_size, **kwargs):
+        return BatchGenerator(self, batch_size = batch_size, mltype=self._vset.mltype, **kwargs)
```
Comment (rjpolackwich) [inline pyveda/vedaset/stream/vedastream.py]: batch_size=batch_size (no spaces)

File: pyveda/frameworks/augmentation_generator.py [hunk 2]
```diff
@@ -78,6 +78,57 @@ def random_rotation_f(x, rg, row_index=1, col_index=2, channel_index=0,
...
+            augmentation_list.append(np.flipud)
+        return augmentation_list
+
+    def on_epoch_end(self):
+        '''update index for each epoch'''
+        self.indexes = np.arange(len(self.list_ids))
+        if self.shuffle:
+            np.random.shuffle(self.indexes)
+
 class BatchGenerator(): #keras.utils.Sequence
```
Comment (chelm) [inline pyveda/frameworks/augmentation_generator.py]: @martham93 im confused why there are 2 class defs for BatchGenerator, possible bad merge?
